db/migrations/1642090824185-seed-report-data.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class SeedReportData1642090824185 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+        INSERT INTO field_value ("reportId", "fieldId", "value", "version")
+        VALUES ('0a15ce57-c46e-4b71-84f0-49dbed4bb81e', 1, 'Lenore Marquez', 1),
```

Comment:
This is adding dummy data into this particular Oral Report, this will happen as part of the integration with NDelius in the future.